packages/shared/profiles/sagas.ts

```diff
@@ -95,7 +96,7 @@ export function* profileSaga(): any {
   yield takeEvery(USER_AUTHENTIFIED, initialProfileLoad)
 
   yield takeLatestByUserId(PROFILE_REQUEST, handleFetchProfile)
-  yield takeLatestByUserId(PROFILE_SUCCESS, submitProfileToRenderer)
+  yield takeLatestByUserId(PROFILE_SUCCESS, submitProfileToRendererAndScenes)
```

Comment:
I think you should hook to `SAVE_PROFILE_SUCCESS` (after self profile changed) instead of `PROFILE_SUCCESS` (after all avatars either load or change)

Reply:
ok, I thought we wanted to know when every profile changes and even load. For example if I'm running the scene A and this scene wants to match me with users that don't have any portable experience. It will only know my changes with SAVE_PROFILE_SUCCESS